src/istgt.c

```diff
@@ -2633,6 +2643,38 @@ void *timerfn(void *ptr __attribute__((__unused__)))
 		}
 		while((conn = (CONN *)(istgt_queue_dequeue(&backupconns))) != NULL)
 			istgt_queue_enqueue(&closedconns, conn);
+
+		clock_gettime(clockid, &now);
+		timesdiff(clockid, last_check, now, diff);
+		ms = diff.tv_sec * 1000;
+		ms += diff.tv_nsec / 1000000;
+
+		if (ms > check_interval1) {
+			check_interval1 = check_interval2;
```

Comment:
why do we need check_interval2? can you put some comments in the logic?
Also, put some comments in data_conn.c for logic.

Reply:
ok

Reply:
`check_interval2` can be removed.